docs/data/charts/zoom-and-pan/ZoomSlider.tsx

```diff
@@ -24,7 +24,7 @@ export default function ZoomSlider() {
             filterMode: 'discard',
             minSpan: 10,
             panning: true,
-            slider: { enabled: true },
+            slider: { enabled: true, showTooltip: 'hover' },
```

Comment:
It is needed here? 🤔

Reply:
Nope, it's a leftover from when I was testing. Good catch!